index.html

```diff
@@ -80,7 +80,7 @@ <h2>Stages</h2>
     </td>
     <td>None
     <td>Major
-    <td>Polyfills / demos
+    <td>Reference implemnentations / demos
```

Comment:
```suggestion
   <td>Demos and experimental reference implementations (implementations that modify global objects should not be published as libraries or used in production-deployed code)
```

Reply:
I also think the “Experimental” in Stage 2 should be further clarified to: “Experimental engine implementations and reference implementations (implementations that modify global objects should not be published as libraries or used in production-deployed code)”.